controllers/resource/resource_generator.go

```diff
@@ -35,6 +35,7 @@ const (
 const (
 	doguPodNamespace = "POD_NAMESPACE"
 	doguPodName      = "POD_NAME"
+	doguPodMultiNode = "POD_MULTI_NODE"
```

Comment:
Maybe somthing like `CLOUDOGU_MULTINODE` would be more precise and safer for collisions.

Reply:
or `ECOSYSTEM_MULTINODE`